Include/internal/pycore_frame.h

```diff
@@ -44,7 +46,7 @@ typedef struct _interpreter_frame {
     int f_lasti;       /* Last instruction if called */
     int stacktop;     /* Offset of TOS from localsplus  */
     PyFrameState f_state;  /* What state the frame is in */
-    int depth; /* Depth of the frame in a ceval loop */
+    bool own_cframe;  // Whether this is the "root" frame for the current CFrame
```

Comment:
I'm not sure about this name.

I don't think the frame really owns the C frame. Also, be wary of using CFrame, as `CFrame` is a struct name.
If I were to suggest a name, it would be `is_entry` as this is the entry `InterpreterFrame` for `_PyEval_EvalFrameDefault`.
If you think ownership is a better mental model, then `owns_cframe` is fine.

Reply:
@markshannon Can it be named `backed_by_cframe` or `cframe_backed` instead?

Reply:
This has nothing to do with how the data is stored.

Reply:
I agree with Mark, I think it should be called `entry_frame` or something similar. Reasoning about ownership is a bit more obscure in my opinion

Reply:
I was hoping somebody would suggest a better name! `is_entry` it is...